docs/CHANGELOG.rst

```diff
@@ -13,6 +13,9 @@ Unreleased
 Added
 -----
 * Add Ubuntu 17.10 base Docker image
+* Add processes: ``meta-alignment``, ``meta-annotation``, ``meta-expression``
```

Comment:
Please extend the description with more information what value does this give to the end user. For instance, that by introducing this processes, we are making sure that all processes of the same type and subprocesses have the output fields that are defined in the meta process. That this makes the web app and ReSDK scripts more robust, because they can expect the same set of outputs on all processes of the same type...

Reply:
I've updated this, let me know what you think.